### PR TITLE
Reorganise electrical: remove ac/dc distinction, branches for equipment types

### PR DESCRIPTION
**Pull request by tkurki**
This PR organises electrical schemas so that
- there is just one electrical schema, no ac/dc separation
- schema has definitions for [dcQuantities](https://github.com/SignalK/specification/blob/88499feef8da0ca334980f582b0ba3f553373028/schemas/groups/electrical.json#L52) and [acQuantities](https://github.com/SignalK/specification/blob/88499feef8da0ca334980f582b0ba3f553373028/schemas/groups/electrical.json#L180)
- equipment types are branches with keyed instances
- equipment types are composed by adding ac, dc or both quantities and adding any additional properties such as [battery chemistry](https://github.com/SignalK/specification/blob/88499feef8da0ca334980f582b0ba3f553373028/schemas/groups/electrical.json#L262)

See also #183

### CODE REVIEW

**Comment by tkurki** [comment]
There is stuff to do like
- [ ] add charger
- [ ] add solar panel
but should this be merged and then additional equipment added as separate, smaller PRs - are we happy with this organisation or should it be stressed with more sample configurations, the one I came up with is still pretty simplistic?

File: schemas/groups/electrical.json [hunk 1]
```diff
@@ -177,32 +177,73 @@
...
+        },
+        "reactivePower": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "Reactive power",
+          "units": "W"
+        },
+        "powerFactor": {
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "description": "Power factor",
+          "unit": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX fraction"
```
Comment (tkurki) [inline schemas/groups/electrical.json]: @canboat what do we use for unit here?

Comment (canboat) [inline schemas/groups/electrical.json]: We have `ratio` with description 'Relative value compared to reference or normal value. 0 = 0%, 1 = 100%, 1e-3 = 1 ppt' which fits perfectly for this.

**Comment by tkurki** [comment]
I think this is ripe for merging, additional equipment can be added as needed.

**Comment by sumps** [comment]
In these Mapped objects types like Electrical, Tanks, etc. is there any reason why you would not call the object /battery.1 rather than as you have it at the moment /battery/1 ? From a REST API point of view would it not be better to call /electrical/ and get a list returned of battery.1, battery.2, inverter, etc rather than have an extra branch for each object ?

**Comment by timmathews** [comment]
I've made a few changes, pretty minor stuff. The biggest change was using plain "number" as the type for the temperature and current limit values

**Pull request merged**